pkg/crypto/tinkcrypto/primitive/cl/subtle/test_utils.go

```diff
@@ -85,16 +95,21 @@ func Verify(
 	verifier, err := ursa.NewProofVerifier()
 	require.NoError(t, err)
 
```

Comment:
As we have a verifier entity now, maybe we can remove the `Verify` methods from `test_utils` and use the `CLVerifier` instead in tests?